src/dune_rules/lib.ml

```diff
@@ -474,6 +478,7 @@ let equal : t -> t -> bool = phys_equal
 let hash = Poly.hash
 
 include Comparable.Make (T)
+module Tbl = Hashtbl.Make (T)
```

Comment:
To avoid duplicating code, you can put the lines with `equal` and `hash` inside a module `H`, and then `include H` and `module Tbl = Hashtbl.Make(H)`.